xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/XWiki.java

```diff
@@ -3396,7 +3396,12 @@ public int createUser(boolean withValidation, String userRights, XWikiContext co
             // Send validation mail, if needed.
             if ((result > 0) && (withValidation)) {
                 // Send the validation email
-                sendValidationEmail(xwikiname, password, email, validkey, "validation_email_content", context);
+                try {
+                    sendValidationEmail(xwikiname, password, email, validkey, "validation_email_content", context);  
+                } catch (XWikiException e) {
+                    LOGGER.error("User created. Failed to send the email to the created user", e);
```

Comment:
This is not an error. ERROR means not recoverable. In this case it's a WARN and you should display a stack trace. You should use ExceptionUtils to get the root message. See other example.

I'm pretty sure I had documented this on dev.xwiki.org but can't find it ATM.